tests/OSPSuite.DataBinding.DevExpress.Starter/FormTest.cs

```diff
@@ -87,7 +87,7 @@ private void InitBinding()
          _screenBinder.Bind(item => item.Value).To(textEditDouble);
          _screenBinder.Bind(item => item.Log).To(memoEdit);
 
-         _screenBinder.Bind(item => item.NullableValue).To(textNullableEdit);
+         _screenBinder.Bind(item => item.IntValue).To(textNullableEdit);
```

Comment:
I did at first. But then I could not build anymore. All the references to Devexpress are broken.
I can continue to try and figure that out, revert the change, or leave it as-is.

Something weird when I use the designer on that form.